stf/tasks/split_forecast_into_components.py

```diff
@@ -33,7 +33,7 @@
 
 def main():
     with TaskContext(__file__) as context:
-        model_type = "xgb"
+        model_type = "lgb"
 
```

Comment:
xgb shoud remain xgb but lgb should be added to the list

Reply:
I'll change it, thanks!

Reply:
This is not changed and merged in develop it seems. This does break our tasks I guess?